README.md

```diff
@@ -166,6 +166,7 @@ Run `lerna --help` to see all available commands and options.
 - `command.publish.message`: a custom commit message when performing version updates for publication. See [@lerna/version](commands/version#--message-msg) for more details.
 - `command.publish.registry`: use it to set a custom registry url to publish to instead of
 npmjs.org, you must already be authenticated if required.
+- `command.publish.changelogHeader`: use it to set a custom change log header.
```

Comment:
This should actually be in the `command.version` namespace, as it is `lerna version` that creates the changelog entries (called implicitly from `lerna publish` in 3.x).